src/components/organisms/TimingSpace/TimingSpace.scss

```diff
@@ -0,0 +1,42 @@
+@import "scss/constants.scss";
+
+$create-button-width: 136px;
+
+.TimingSpace {
+  margin: 0 $spacing--xxl $spacing--xl;
+
+  &__header {
+    cursor: pointer;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__name {
+    font-weight: $font-weight--500;
+    font-size: $font-size--lg;
+    line-height: 19px;
+  }
+
+  &__title {
+    font-size: $font-size--md;
+    line-height: 17px;
+    color: opaque-white(0.7);
+    margin-left: $spacing--sm;
+  }
+
+  &__createButton {
+    max-width: $create-button-width;
+    font-weight: $font-weight--500;
+    font-size: $font-size--md;
+    line-height: 17px;
+  }
+
+  &__content {
+    margin-top: $spacing--md;
+
+    & > .TimingEvent:nth-of-type(odd) {
+      background-color: #19181a;
```

Comment:
`constants.scss`

Reply:
fixed in 694c85c4c23d817716346efdbc1df16c7b903d58